sdk/digitaltwins/azure-digitaltwins-core/API design.md

```diff
@@ -726,9 +726,159 @@ When updating a model, the payload for a multi-operation json patch follows the
 
 <details><summary><b>APIs</b></summary>
 
+Async APIs
+
+```java	
+    /**
+     * Creates one or many models synchronously.
```

Comment:
Did you mean to say asynchronously here and below?

Reply:
yes yes